test/index.js

```diff
@@ -56,7 +64,8 @@ test('validate-npm-package-name', function (t) {
   t.deepEqual(validate(' leading-space'), {
     validForNewPackages: false,
     validForOldPackages: false,
-    errors: ['name cannot contain leading or trailing spaces', 'name can only contain URL-friendly characters']})
+    errors: ['name cannot contain leading or trailing spaces', 'name can only contain URL-friendly characters'],
+    warnings: ['name cannot start with a special/capital character']})
 
```

Comment:
Lets add another test case of a trailing whitespace if not already present.

Reply:
We do have a test case for that condition.